src/report.ts

```diff
@@ -47,16 +51,19 @@ export class ReportModule {
         exportConfig: IExportConfig = {},
         pollingOptions: IPollingOptions = {},
     ): Promise<IExportResponse> {
-        const showFilters: AFM.CompatibilityFilter[] = exportConfig.showFilters
-            ? compact(exportConfig.showFilters.map(this.convertFilter))
-            : [];
+        const { showFilters } = exportConfig;
+
+        let sanitizedFilters: ExecuteAFM.CompatibilityFilter[] | undefined;
```

Comment:
move this to the separate function sanitizeFilters(showFilters: ExecuteAFM.CompatibilityFilter[]) {
        if (isArray(showFilters) && showFilters.length > 0) {
            return compact(showFilters.map(this.sanitizeFilter));
        }
        return undefined;
}

to split the code and get rid of 'let' :)

Reply:
updated